src/Repository/VehicleRepository.php

```diff
@@ -19,22 +19,30 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Vehicle::class);
     }
 
-    // /**
-    //  * @return Vehicle[] Returns an array of Vehicle objects
-    //  */
-    /*
-    public function findByExampleField($value)
+    /**
+    * @return Vehicle[] Returns an array of Vehicle objects
+    */
+    public function findByParticularVehicle(): array
     {
         return $this->createQueryBuilder('v')
-            ->andWhere('v.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('v.id', 'ASC')
-            ->setMaxResults(10)
+            ->andWhere('v.type = :ParticularVehicle')
```

Comment:
What's the point of having a parameter when in fact you don't have parameter ?
And if you want vehicule by type, you don't need specific queries findBy should be enough

Reply:
an oversight on my part, it was a test.
I forgot to remove this part